### DISCUSSION TITLE
Is this project still being monitored by the mods?

### DISCUSSION THREAD

**Discussion by rick2909** [UI-Lovelace-Minimalist/UI#1473] [General]
We can't merge pull request so we can't fix issues that come up due to HA updates. or even add new card and or functions.

**Comment by edoardop13** [comment]
I think after the release of mushrooms it isn't maintained. With mushrooms you can create your custom template like this project

**Reply by rick2909** [reply]
true but it is still missing alot of the amazing custom cards that are in this project. it would be a shame to just leave it.

**Reply by billyjoebob999** [reply]
Starting to feel like that way. I posted an issue back in October but nothing. I ended up fixing it myself a couple months later BUT since I was mostly using mushroom cards anyways I decided to completely recreate my dashboards using mushroom. Hopefully the dev comes back soon or some else is able to pick this up.
